clients/wallet/src/keys.rs

```diff
@@ -7,7 +7,11 @@ use std::env;
 // previously passed to the environment.
 // A variable not .env but passed to the environment will not be overridden.
 fn get_env_variables(key: &str) -> Option<String> {
-	dotenv::from_filename("../.env").ok();
+	if let None = dotenv::from_filename("../.env").ok() {
```

Comment:
had a problem with running the example in `stellar-relay-lib`.
I moved the env file to the root directory; that's why I added a condition to try looking at the current directory.